cloudformation/media-atom-maker.json

```diff
@@ -55,6 +55,14 @@
         "AuditTable": {
             "Description": "Name of the audit dynamo table",
             "Type": "String"
+        },
+        "AlertWebhook": {
+          "Description": "Where CloudWatch alerts are sent",
+          "Type": "String"
+        },
+        "AlertActive": {
```

Comment:
ah ok. It'll probably be best to keep all alarms on for now. We can revisit later once the app has proven stability.